cfg/1.11/node.yaml

```diff
@@ -26,17 +26,19 @@ groups:
       systemctl daemon-reload
       systemctl restart kubelet.service
     scored: true
-
   - id: 2.1.2
```

Comment:
Not a giant deal but I think it's a shame for readability to lose the blank lines like the one that was at line 29